pkg/network/ebpf/c/defs.h

```diff
@@ -1,6 +1,10 @@
 #ifndef _DEFS_H_
 #define _DEFS_H_
 
+#ifndef __maybe_unused
```

Comment:
Can't use the one defined here: https://github.com/DataDog/datadog-agent/blob/main/pkg/ebpf/c/compiler.h#L7?